R/utils.R

```diff
@@ -258,7 +247,19 @@ rowcol_to_dims <- function(row, col) {
   stringi::stri_join(min_col, min_row, ":", max_col, max_row)
 
 }
+#' @rdname dims_helper
+#' @noRd
```

Comment:
Maybe export this function. See `dims_helper.Rd` to see the extent of changes here.

Reply:
this is only for internal useage. The only difference should be that `_dims(1,1)` will return `A1:A1` and `_dim(1,1)` will return `A1`. It matters internally, because we have a few cases, where we use `strsplit()`. Obviously this calls for internal cleanups, but... maybe there are more important things to do

Reply:
Good to know